packages/openapi-typescript/test/helpers.ts

```diff
@@ -1,12 +0,0 @@
-import fs from "node:fs";
-
-export const CLRF_RE = /\r\n/g;
-
-/** Normalize all linebreaks to \n when reading files (for Windows tests) */
-export function readFile(filepath: fs.PathOrFileDescriptor): string {
```

Comment:
Turns out this is built in to Vitest. So now we just use `.toMatchFileSnapshot()` instead

Sidenote for readers: I don’t like “traditional” snapshots for 2 reasons:
1. I believe snapshots shouldn’t have any test runner artifacts/comments and should be 100% human-readable (otherwise, you miss changes)
2. I don’t like making snapshots easy to update in a project like this—the snapshots are everything. I want failures to be meaningful, and require investigation into the _why_ something changed, and to not disappear with the simple addition of a `-u` flag.